### DISCUSSION TITLE
<CR> to open file in split

### DISCUSSION THREAD

**Discussion by yngwi** [NeogitOrg/neogit#462] [Q&A] [answered]
Hi, is it possible to change the `<CR>` mapping on a file to open it in a split in the Neogit tab instead of closing the Neogit tab and open the file in the original window? Before commits I often go again to several of the changed files to see if they still have any linter warnings and then directly return to the Git status window.

**Comment by CKolkey** [comment, marked as answer]
Have a look here: https://github.com/TimUntersberger/neogit/blob/7be1e9358aaa617b0391e61952d936203e99fcf0/lua/neogit/status.lua#L896

I believe the specific command you're after is on line 919. Currently not possible as it's hardcoded to be `vim.cmd("e " .. filepath)` _but_ you could probably work out a way to do it.

**Reply by yngwi** [reply]
Thank you
